docs/topics/spiders.rst

```diff
@@ -158,7 +158,7 @@ scrapy.Spider
        If you want to change the Requests used to start scraping a domain, this is
        the method to override. For example, if you need to start by logging in using
        a POST request, you could do::
-
+            
```

Comment:
Please, make sure you are not making unintended changes. Review your own changes in GitHub before marking your pull request for review.

Reply:
ok